### ISSUE TITLE
Relationship between tutorial version and documentation version switcher

### ISSUE THREAD

**Issue by kandarpksk** [napari/docs#17]
## 📚 Documentation

There may be other pages/tutorials, but I've only noticed the one I'm using as an example here: the [quick start tutorial/page](https://napari.org/stable/tutorials/fundamentals/quick_start.html) says that it uses napari v0.4.14, which seems a bit off now that we have a documentation version switcher which defaults to the newest stable version, currently 0.4.16. Maybe it's fine as-is, but maybe there's something better we can do here.

Specific to the quick start page, it also links to download v0.4.14 of the app and suggests downloading the "latest" release from the GitHub releases page. The word "release" has a different meaning here (I think) than in the version switcher.

**Comment by melissawm** [comment]
Hi folks, on this thought I was also thinking about how it could be useful to have a way to build and check the narrative documentation without needing to download or clone the main repo. Right now this is possible if we install napari from any source (pip, conda) and build the docs with

```
$ make html-noplot  # doesn't build the gallery
```

This would list the API docs for the version currently installed on this environment/system. (This still requires the examples folder in the right location though. Ideally, we could also have a way to disable the gallery entirely when running this workflow)

As a proof of concept, I've amended the API docs index page to print the version for which the docs are rendered, so we have less potential for confusion. Thoughts?

cc @jni

**Comment by melissawm** [comment]
Coming back to this, since our tutorials are mostly autogenerated, I'm not sure this is relevant anymore. Maybe what we want is a regular (every release) check of the screenshots to make sure they are still current. Thoughts?

**Comment by jni** [comment]
> Maybe what we want is a regular (every release) check of the screenshots to make sure they are still current. Thoughts?

Yes this is a good short term solution. Long term as we have discussed elsewhere, it would be great if we used pyautogui or similar to actually run the tutorial pages on each build and automatically generate the screenshots.
